backend/src/main/resources/application.yml

```
@@ -23,6 +23,9 @@ spring:
 dataloader:
   enable: true
 
+apply:
+  post:
+    base-url: https://local.cruru.kr/post/%d
```

```suggestion
    base-url: https://www.cruru.kr/post/%d
```

`local.cruru.kr` URL은 실제하지 않는 URL이니 실제 프로덕션 기준으로 작성해도 좋을 것 같습니다

---

좋은 생각입니다. 반영하겠습니다.